tests/designs/uart2bus/Makefile

```diff
@@ -27,10 +27,10 @@ ifeq ($(SIM),aldec)
     GPI_EXTRA=vhpi
 endif
 ifeq ($(SIM),modelsim)
-    GPI_EXTRA=fli
```

Comment:
@stuarthodgson I submitted issue #372, to address this.  I would prefer not to change the questa makefile at this time because we need to determine all makefiles that need to be updated to support this.  Further in that issue I have proposed a solution that will remove the need for this all together.